File: troc-11/troc-11-raja/solution-rama.cpp
/**
 * Author   : rama_pang
 * Problem  : troc-11-raja
 * Time     : O(1)
 * Memory   : O(1)
**/

#include <bits/stdc++.h>
using namespace std;

int main() {
  int N, M;
  cin >> N >> M;

  long long ans = 0;

  ans += 8ll * (N - 2); // corners on side N
  ans += 8ll * (M - 2); // corners on side M

  ans += 2ll * (N - 2) * (N - 3); // side N
  ans += 2ll * (M - 2) * (M - 3); // side M
  
  cout << ans << "\n";
  return 0;
}

File: troc-12/troc-12-dunia/solution_rama.py
# Author   : rama_pang
# Problem  : troc-12-dunia
# Time     : O(1)
# Memory   : O(1)

from sys import stdin, stdout

def main():
  A, X = [int(x) for x in stdin.readline().split()]
  stdout.write(str(X // A) + ' ' + str(X % A) + '\n')

if __name__ == '__main__':
  main()